index.bs

```diff
@@ -1082,6 +1082,39 @@ spec: webidl
       permission is the permission associated with the usage of the
       asynchronous methods in the [[clipboard-apis]].
     </p>
+    <dl>
+      <dt>
+        <a>permission descriptor type</a>
+      </dt>
+      <dd>
+        <pre class='idl'>
+          enum ClipboardPermissionAccess {
+            "full",
+            "write",
+          };
+          
+          dictionary ClipboardPermissionDescriptor : PermissionDescriptor {
+            ClipboardPermissionAccess access;
```

Comment:
I wonder if this should have the default ` ClipboardPermissionAccess access = "write"`? It takes the ambiguity out of what it means for it to be missing (which implies "write").